lib/utils/ens/index.ts

```diff
@@ -0,0 +1,16 @@
+import { getDefaultProvider } from "@ethersproject/providers";
+
+export const isENSDomain = (domain: string) =>
+  !!domain && domain.includes(".eth");
+
+export const ETHProvider = getDefaultProvider();
```

Comment:
Ah interesting stuff. Did not know this.

I refactored all places and pass now the Infura URL to the provider instead of the ID.
This way we can pass different Urls around.

There are no further occurrences of `getDefaultProvider` on the server.
Only in the App on the client.

Please check again